serving/docker/pytorch-gpu.Dockerfile

```diff
@@ -44,7 +46,7 @@ ENV PYTORCH_FLAVOR=cu124-precxx11
 ENV TORCH_CUDNN_V8_API_DISABLED=1
 ENV JAVA_OPTS="-Xmx1g -Xms1g -XX:+ExitOnOutOfMemoryError -Dai.djl.default_engine=PyTorch"
 ENV HF_HOME=/tmp/.cache/huggingface
-ENV TRANSFORMERS_CACHE=/tmp/.cache/huggingface/transformers
```

Comment:
@xyang16 Don't we need TRANSFORMERS_CACHE anymore?

Reply:
This is deprecated, only need HF_HOME